bugbug/model.py

```diff
@@ -204,20 +205,40 @@ def train(self, importance_cutoff=0.15):
 
             matplotlib.pyplot.savefig("feature_importance.png", bbox_inches="tight")
 
-            # TODO: Actually implement feature importance visualization for multiclass problems.
+            avg_shap_values = 0
             if isinstance(shap_values, list):
-                shap_values = np.sum(np.abs(shap_values), axis=0)
+                avg_shap_values = np.sum(np.abs(shap_values), axis=0)
+            else:
+                avg_shap_values = shap_values
+                shap_values = [shap_values]
 
             important_features = self.get_important_features(
-                importance_cutoff, shap_values
+                importance_cutoff, avg_shap_values
             )
 
             print(f"\nTop {len(important_features)} Features:")
-            for i, [importance, index, is_positive] in enumerate(important_features):
-                print(
-                    f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
+            top_indexes = list(
+                int(index) for importance, index, is_pos in important_features
+            )
+
+            table = []
+            for num, item in enumerate(shap_values):
```

Comment:
Part of this should be moved to get_important_features.
get_important_features could return an object like:
```
{
    "average": most important features in average shap values,
    "classes": {
        "class1": (most important features in class1 shap values, values in class1 of the most important features in average shap values),
        "class2": (most important features in class1 shap values, values in class1 of the most important features in average shap values),
    }
}
```

This way here you can show the most important feaures in average shap values and their values for each class, and in classify you can show the most important features for the predicted class.

Reply:
so we want the list of most important features for each class, distinctly, to use in classify but we do not need that here, right?

Reply:
Exactly.
At training time, we will show the most important average features and their values for all classes.
At classify time, we will show the most important features for the predicted class.

get_important_features will return everything though, so we can easily do more complex stuff in the future if we want to.